front_end/panels/ai_chat/ui/SettingsDialog.ts

```
@@ -2319,6 +2421,459 @@ export class SettingsDialog {
+    mcpConfigContainer.appendChild(mcpMaxMcpHint);
+
+    const mcpMaxMcpInput = document.createElement('input');
+    mcpMaxMcpInput.type = 'number';
+    mcpMaxMcpInput.className = 'settings-input';
+    mcpMaxMcpInput.min = '1';
+    mcpMaxMcpInput.max = '50';
+    mcpMaxMcpInput.value = String(currentMCPConfig.maxMcpPerTurn || 8);
+    mcpConfigContainer.appendChild(mcpMaxMcpInput);
+
+    // Handle budget control changes
+    const updateBudgetControls = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)